modules/mtc-operator-supported.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * migration_toolkit_for_containers/about-mtc.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="mtc-operator-supported_{context}"]
+= Support for {mtc-first}
+
+.Supported versions of {mtc-short}
+[width="100%",cols="10%,14%,20%,22%,22%,22%,options="header"]
+|===
+|Version
+|{mtc-short} version
+|General availability
+|Full support ends
+|Maintenance ends
+|Extended Update Support (EUS)
+
+|1.8
+a|
+* 4.14
+* 4.15
+* 4.16
+|02 Oct 2023
+|Release of 1.9 
+|Release of 1.10
+|N/A  
+
+|1.7
+a|
+* 3
```

Comment:
this should be 4.13 right?

Reply:
@rayfordj & @dymurray - Please can I confirm this
 
![image](https://github.com/user-attachments/assets/192adea4-c314-41a8-b567-37079b2a1c8f)

Reply:
> this should be 4.13 right?

While I originally glossed over this catching up on things, I've since had time to reflect and have further perspective to provide... I would expect this to be 4.12, 4.13, 4.14, 4.15, 4.16, & 4.17.  Once 4.18 is GA, this would move to 4.14, 4.15, 4.16, 4.17, 4.18.  When 4.20 is GA, this would again move to reflect 4.16, 4.17, 4.18, 4.19, 4.20.  And so it continues.

From what I recall (and has been implemented) with original MTC 1.8 discussions, it was our intention to support two prior EUS versions on a sliding window.   For OCP v4.16/v4.17, that means MTC 1.8 support on 4.12, 4.13, 4.14, 4.15, 4.16, & 4.17.  When 4.18 (& 4.19) are GA, that moves to MTC 1.8 support on 4.14, 4.15, 4.16, 4.17, 4.18, & 4.19.  If this position has changed, we should stop shipping 1.8 to the EOL EUS and their +1 odd release.  If we only ship/support 1.8 on non-EOL OCP versions, it would allow us to adjust `com.redhat.openshift.versions=` to automatically ship MTC 1.8 to non-EOL OCP versions as they're created.  Today, we must manually track & update this to cover the necessary OCP versions supported.

@dymurray , need you to weigh in here on the official position for MTC 1.8.  
Whatever the outcome, we need to ensure we update the docs appropriately -- MTC docs and the support policy.  Additionally, this will dictate where we ship future 1.8 releases -- continue supporting & shipping MTC 1.8 updates on two previous EUS versions or not.

Reply:
:thought_balloon::  _Perhaps rather than tracking OCP latest-minus-some-number version, we simply move to "supported on non-EOL OCP versions", set `com.redhat.openshift.versions=` accordingly, update docs to reflect appropriately, and let it ride?_

Reply:
OCP [Life Cycle Dates](https://access.redhat.com/support/policy/updates/openshift/#dates)